### PR TITLE
dev-python/vdirsyncer-0.16.8: bump

### PR DESCRIPTION
**Pull request by Hawk777**

### CODE REVIEW

**Comment by Hawk777** [comment]
Rebased and transformed this from a Python support PR into a version bump and EAPI bump PR, since 0.16.8 has now been released.

File: dev-python/vdirsyncer/vdirsyncer-0.16.8.ebuild [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
 DESCRIPTION="Synchronize calendars and contacts"
 HOMEPAGE="https://github.com/pimutils/vdirsyncer"
 SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
-LICENSE="MIT"
+LICENSE="BSD"
 KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 SLOT="0"
-IUSE="test"
-RESTRICT="!test? ( test )"
```
Comment (epsilon-0) [inline dev-python/vdirsyncer/vdirsyncer-0.16.8.ebuild:19]: use `distutils-r1_enable_tests pytest`  
https://dev.gentoo.org/~mgorny/python-guide/distutils.html#enabling-tests

A lot of things to change when you read that whole page on tests

File: dev-python/vdirsyncer/vdirsyncer-0.16.8.ebuild [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+
+PATCHES=( "${FILESDIR}/click-compat.patch" )
```
Comment (juippis) [inline dev-python/vdirsyncer/vdirsyncer-0.16.8.ebuild:20]: Please include the version, `PV`, when this patch was added.

`vdirsyncer-0.16.8-click-7-compat.patch` or similar.

Comment (Hawk777) [reply dev-python/vdirsyncer/vdirsyncer-0.16.8.ebuild]: 28fe50a7865c0d14e9e17d7f6975c8f95eb8f15a renames the patch file

**Comment by Hawk777** [comment]
> Split this into two commits. One of adding a new file, 2nd commit for removing old version.

Removing the old commit split out into a1598c3bfb6d7752288d47e8727ad7de495a4ce7.

**Pull request closed without merging**